client/src/app/+videos/+video-watch/video-watch.component.ts

```diff
@@ -219,6 +219,12 @@ export class VideoWatchComponent implements OnInit, OnDestroy {
     return $localize`You need to be <a href="/login">logged in</a> to rate this video.`
   }
 
+  getExactNumberOfViews () {
+    return (this.video.views >= 1000)
+      ? (this.video.views + ' ' + (this.video.isLive ? 'viewers' : 'views'))
```

Comment:
```suggestion
      ? (this.video.views + ' ' + (this.video.isLive ? $localize`viewers` : $localize`views`))
```

Reply:
oh nice, I've not use` $localize` until now, but it's so cool. Thks, I make change :+1: